File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    max_width = 0
    max_height = 0
    for size in sizes:
        size = sorted(size)
        if size[0] > max_width:
            max_width = size[0]
        if size[1] > max_height:
            max_height = size[1]
    return max_width * max_height

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.swift
import Foundation

func solution(_ sizes:[[Int]]) -> Int {
    var sortedsize = sizes.map {$0.sorted(by:>)}
    
    var width: Set<Int> = []
    var height: Set<Int> = []
    for size in sortedsize {
        width.insert(size[0])
        height.insert(size[1])
    }
    let answer = width.max()! * height.max()!
    return answer
}